AccessCheckoutDemo/AccessCheckoutDemoUITests/utils/TestUtils.swift

```diff
@@ -3,6 +3,7 @@ import XCTest
 struct TestUtils {
     // 25 attempts over 5 seconds
     private static let assertCardBrandMaxAttempts = 25
+    private static let assertLabelMaxAttemps = 50
     private static let assertCardBrandSleeptBetweenAttemptsInMs = 0.2
```

Comment:
can we rename this since the sleep time is now common between the assertCard assertLabel functions?